obspy/signal/tests/test_seismicArray.py

```diff
@@ -0,0 +1,79 @@
+from unittest import TestCase
+
+from obspy.signal.array_analysis import SeismicArray, correct_with_3dplane
+from obspy.core.inventory.station import Station
+from obspy.core.inventory.network import Network
+import numpy as np
+
+
+class TestSeismicArray(TestCase):
+    """
+    Test cases for the Seismic Array class.
+    """
+
+    def setUp(self):
+        codes = "bl br m ul ur".split()
+        lat = [0, 0, 1, 2, 2]
+        long = [0, 2, 1, 0, 2]
+        elev = [0, 0, 0, 0, 0]
+        stns = [Station(codes[i], lat[i], long[i], elev[i]) for i in
+                range(len(codes))]
+        self.simpleinv = [Network("5pt", stations=stns)]
+
+        #todo: this:
+        self.simpleinv_nochannels = [[[]]]
+
+        self.testarray = SeismicArray()
+        self.testarray.add_inventory(self.simpleinv)
+
+        self.geo_exp = {'5pt.bl': {'absolute_height_in_km': 0.0,
```

Comment:
This is the same data as `self.simpleinv`, right? I think it should be generated from it instead of duplicated here.